tree/ntuple/v7/inc/ROOT/RPageStorageDaos.hxx

```diff
@@ -115,8 +115,8 @@ struct RDaosContainerNTupleLocator {
    static ntuple_index_t Hash(const std::string &ntupleName)
    {
       // Convert string to numeric representation via `std::hash`.
-      std::size_t h = std::hash<std::string>{}(ntupleName);
-      // Fold `std::size_t` bits into 32-bit using `boost::hash_combine()` algorithm and magic number.
+      auto h =  static_cast<uint64_t>(std::hash<std::string>{}(ntupleName));
```

Comment:
I find this slightly more readable; in both cases, the compiler widens the representation to 64-bit if `sizeof(size_t) < sizeof(std::uint64_t)`.
```suggestion
      std::uint64_t h =  std::hash<std::string>{}(ntupleName);
```